ui/app/components/auth/form/base.ts

```diff
@@ -52,41 +54,41 @@ export default class AuthBase extends Component<Args> {
   login = task(
     waitFor(async (formData) => {
       try {
-        const authResponse = await this.auth.authenticate({
-          clusterId: this.args.cluster.id,
-          backend: this.args.authType,
-          data: formData,
-          selectedAuth: this.args.authType,
-        });
-
-        const path = formData?.path;
-        this.handleAuthResponse(authResponse, path);
+        const normalizedAuthData = await this.loginRequest(formData);
```

Comment:
this parent class calls`this.loginRequest` which is defined by each child component and calls the relevant login request